tools/azsdk-cli/Azure.Sdk.Tools.Cli.Tests/Helpers/LabelHelperTests.cs

```diff
@@ -0,0 +1,98 @@
+using Azure.Sdk.Tools.Cli.Helpers;
+using Azure.Sdk.Tools.Cli.Tests.TestHelpers;
+
+
+namespace Azure.Sdk.Tools.Cli.Tests.Helpers;
+
+internal class LabelHelperTests
+{
+    private ILabelHelper labelHelper;
+    private TestLogger<LabelHelper> testLogger;
```

Comment:
Remove `private TestLogger<LabelHelper> testLogger;` ... It's not being used anywhere.